src/main/scala/nl/knaw/dans/easy/multideposit/actions/AddDatasetMetadataToDeposit.scala

```diff
@@ -167,6 +167,7 @@ object AddDatasetMetadataToDeposit {
         case (Some(q), None,   Some(t)) => elem(s"dcterms:$q")(t)
         case (None,    Some(l),_      ) => elem(s"dc:relation")(l)
         case (None,    None,   Some(t)) => elem(s"dc:relation")(t)
+        case _                          => // TODO do nothing; this needs to be checked in the preconditions
```

Comment:
It is certainly good to fix the build warning, but I am not really in favour of keeping TODO's around in the code. They tend not to get fixed. If it is not too complex let's just find out what is appropriate to do here:

1. Throw an `IllegalStateException` (if we should never end up here)
2. Do nothing (if it is possible to legitimately end up here)